ci-operator/step-registry/monitoring-plugin/tests/perses-ui-ivt/monitoring-plugin-tests-perses-ui-ivt-commands.sh

```
@@ -0,0 +1,142 @@
+        wait_auth=false
+    else
+        sleep 10
+    fi
+done
+echo "authentication operator finished updating"
+
+# Copy the artifacts to the aritfact directory at the end of the test run.
+trap copyArtifacts EXIT
+
+# Set Kubeconfig var for Cypress.
+cp -L $KUBECONFIG /tmp/kubeconfig && export CYPRESS_KUBECONFIG_PATH=/tmp/kubeconfig
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)